content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,361 @@
+the newest lot numbers will be reserved thanks to the :abbr:`LIFO (Last In, First Out)` strategy.
+All four cinder blocks from lot 0000400 and one cinder block from lot 000030 will be selected to be
+sent to the customer.
 
 .. image:: removal/reserved-lots-LIFO.png
    :align: center
-   :alt: two lots being reserved for sell with the LIFO strategy.
+   :alt: Two lots being reserved for sale with the LIFO strategy.
+
+FEFO (First Expired, First Out)
+-------------------------------
 
-.. important::
-   Don’t forget that the *LIFO* strategy is banned in many countries!
+The :abbr:`FEFO (First Expired, First Out)` removal strategy differs from the :abbr:`FIFO (First In,
+First Out)` and :abbr:`LIFO (Last In, First Out)` strategies, because it targets products for
+removal based on **expiration dates** instead of their warehouse receipt dates. With this method,
+the products that are going to expire first are moved out first. This method is used for perishable
+products, such as medicine, food, and beauty products.
 
-FEFO (First Expiry, First Out)
-------------------------------
+Lots are picked based on their **removal date** from earliest to latest. Removal dates indicate how
+many days *before* the expiration date the product needs to be removed from stock. The removal date
+is set on the product form. Lots without a removal date defined are picked after lots with removal
+dates.
 
-With the *FEFO* strategy, the way products are picked is not based on the reception date. In this
-particular case, they are dispatched according to their expiration date.
+.. warning::
+   If products are not removed from stock when they should be, lots that are past the expiration
+   date may still be picked for delivery orders!
 
 .. note::
-   To have more information about Expiration date, please have a look at
-   :doc:`the related doc <../../management/lots_serial_numbers/expiration_dates>`.
+   For more information about expiration dates, reference the
+   :doc:`Expiration dates <../../management/lots_serial_numbers/expiration_dates>` document.
 
-By activating *Expiration Dates*, it becomes possible to define different dates on the serial/lot
-numbers to be used in *FEFO*. These dates can be set by going to :menuselection:`Inventory -->
-Master Data --> Lots/Serial Numbers`.
+First, go to :menuselection:`Inventory app --> Configuration --> Settings` and ensure
+:guilabel:`Expiration Dates` is enabled. Once the :guilabel:`Expiration Dates` setting is enabled,
+it's possible to define different expiration dates for individual serialized products, as well as
+for lot numbers containing many products.
 
-.. image:: removal/removal-date.png
-   :align: center
-   :alt: view of the removal date for 0000001.
+In this example, there are three lots of hand cream. The creams are from the All/Health & Beauty
+category, where :guilabel:`FEFO` is set as the removal strategy. In the :guilabel:`Inventory
```

Comment:
The first half of this example is not part of an active instruction, so using italics rather than guilabel
```suggestion
In this example, there are three lots of hand cream. The creams are from the *All/Health & Beauty*
category, where *FEFO* is set as the removal strategy. In the :guilabel:`Inventory Valuation
```